CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [x.y.z] - yyyy-mm-dd
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
```

Comment:
See the changelog in `manta-rs` for the format (like `Unreleased` instead of `x.y.z`) and also the correct linking setup. Also, for the first `CHANGELOG.md` add at least the previous version changes instead of just nothing.